sign-off-checker/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+name: sign-off-checker
+version: 0.1.0
+description: |
+    A GitHub webhook integration to ensure commits have "Signed-off-by"
+    https://github.com/heptio/sign-off-checker
```

Comment:
nit: to be consistent all yamls in this chart should end with newline (some end with new line, some don't)

That would also help the vim users around to avoid introducing trivial changes...